scripts/azureml-assets/azureml/assets/model/model_utils.py

```diff
@@ -151,3 +143,46 @@ def prepare_model(spec_path, model_config, registry_name, temp_dir):
     except Exception as e:
         logger.log_error(f"prepare model failed for {spec_path}. Error {e}")
         return None, False
+
+
+def update_model_metadata(
+    mlclient: MLClient,
+    model_name: str,
+    model_version: str,
+    update: AssetVersionUpdate,
+):
+    """Update the mutable metadata of already registered Model."""
+    try:
+        model = mlclient.models.get(name=model_name, version=model_version)
+
+        need_update = False
+        if update is not None:  # to update older version of models
+            updated_tags = copy.deepcopy(model.tags)
+            if update.tags:
+                # Replace tags
+                if update.tags.replace is not None:
+                    updated_tags = update.tags.replace
+                elif update.tags.add is not None:
+                    for k, v in update.tags.add.items():
+                        updated_tags[k] = v
+                elif update.tags.delete is not None:
+                    for k in update.tags.delete:
+                        updated_tags.pop(k, None)
+
+            if updated_tags != model.tags:
+                logger.print("tags has been updated.")
+                model.tags = updated_tags
+                need_update = True
+
+            if update.description is not None and model.description != update.description:
```

Comment:
I don't think `update.description` can ever be `None` because it's initialized to the empty string in the class.